client/lib/products-values/index.js

```diff
@@ -210,7 +210,11 @@ export function isDomainProduct( product ) {
 	assertValidProduct( product );
 
 	return (
-		isDomainMapping( product ) || isDomainRegistration( product ) || isPrivacyProtection( product )
+		isDomainMapping( product ) ||
+		isDomainRegistration( product ) ||
+		isPrivacyProtection( product ) ||
+		isDomainTransfer( product ) ||
+		isDomainTransferPrivacy( product )
```

Comment:
Seems like we don't need the check [here](https://github.com/Automattic/wp-calypso/blob/dc3426b6bf30be7eb62734b6d92d043bb3107bcd/client/me/purchases/purchase-item/index.jsx#L154) if transfers in are a domain product.

Reply:
Not the same atm: https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/lib/products-values/index.js#L208

We can review and see if it makes sense to set it as a domain product.